pyiron_atomistics/sphinx/output_parser.py

```diff
@@ -417,3 +418,98 @@ def results(self):
             if len(value) > 0:
                 results["dft"][key] = value
         return results
+
+
+class SphinxWavesParser:
+    """Class to read SPHInX waves.sxb files (HDF5 format)
+
+    Initialize with waves.sxb filename, or use load ()
+
+    """
+
+    def __init__(self, file_name="waves.sxb", cwd="."):
+        """
+        Args:
+            file_name (str): file name
+            cwd (str): directory path
+        """
+        self._eps = None
+        if Path(file_name).is_absolute():
+            self.wfile = h5py.File(Path(file_name))
+        else:
+            path = Path(cwd) / Path(file_name)
+            self.wfile = h5py.File(path)
+
+    @property
+    def _n_gk(self):
+        return self.wfile['nGk'][:]
+    
+    @property
+    def mesh(self):
+        return self.wfile["meshDim"][:]
+
+    @property
+    def Nx(self):
+        return self.wfile['meshDim'][0]
+
+    @property
+    def Ny(self):
+        return self.wfile['meshDim'][1]
+
+    @property
+    def Nz(self):
+        return self.wfile['meshDim'][2]
+
+    @property
+    def n_states(self):
+        return self.wfile["nPerK"][0]
+
+    @property
+    def n_spin(self):
+        return self.wfile["nSpin"].shape[0]
+
+    @property
+    def k_weights(self):
+        return self.wfile["kWeights"][:]
+
+    @property
+    def k_vec(self):
+        return self.wfile["kVec"][:]
+
+    @property
+    def eps(self):
+        """All eigenvalues (in Hartree) as (nk,n_states) block"""
+        if (self._eps is None):
+            self._eps = self.wfile['eps'][:].reshape(-1,self.n_spin,self.n_states)
+        return self._eps.T #change
```

Comment:
```suggestion
    @cached_property
    def eps(self):
        """All eigenvalues (in Hartree) as (nk,n_states) block"""
        return (self.wfile['eps'][:].reshape(-1,self.n_spin,self.n_states)).T
```
The _eps stuff was a manual implementation of what @cached_property does.